lib/cdo/contact_rollups_validation.rb

```diff
@@ -181,8 +166,8 @@ class ContactRollupsValidation
       query: "SELECT COUNT(*) FROM contact_rollups_daily
               WHERE courses_facilitated IS NOT NULL
               AND Roles NOT LIKE '%Facilitator%'",
-      min: 400,
-      max: 40_000
+      min: 0,
+      max: 0
```

Comment:
The name here doesn't match the query. It should specify that it's looking for *non-facilitators* with courses facilitated (which should be 0).